### PR TITLE
📆 Mark current week

### PR DESCRIPTION
**Pull request by anna-wro**
CleanShot 2021-09-07 at 19 35 29@2x

### CODE REVIEW

File: src/utils/getPrayerDataFromFilename.ts [hunk 1]
```diff
@@ -10,9 +10,11 @@ type PrayerDataPropsType = Readonly<{
 
 const PRAYERS = {
   wigilia: { name: 'wigilia', order: 1 },
-  jutrznia: { name: 'jutrznia', order: 2 },
-  nieszpory: { name: 'nieszpory', order: 3 },
-  kompleta: { name: 'kompleta', order: 4 },
+  kompleta_1: { name: 'kompleta I', order: 2 },
```
Comment (anna-wro) [inline src/utils/getPrayerDataFromFilename.ts:13]: I have no idea what to do with Saturday complines. They should appear after the eve, not on Saturday column as we have right now

Comment (anna-wro) [reply src/utils/getPrayerDataFromFilename.ts:13]: Requested by Puchcia

File: package.json [hunk 2]
```diff
@@ -15,6 +15,7 @@
   "dependencies": {
     "classnames": "^2.3.1",
     "gray-matter": "^4.0.3",
+    "luxon": "^2.0.2",
```
Comment (k1c14k) [inline package.json:18]: not 2.0.3 as @types/luxon????

**Review by k1c14k — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (anna-wro) [reply package.json:18]: nope, 2.0.2 is the newest version: https://www.npmjs.com/package/luxon

**Pull request merged**